front/src/modules/simulationResult/components/SpaceTimeChart/SpaceTimeChartHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import { KebabHorizontal } from '@osrd-project/ui-icons';
```

Comment:
Seems like this was added in this PR by mistake?

Reply:
Yep it comes from another PR, not sure how this file ended up here. I just amended the last commit :)